www/index.html

```diff
@@ -53,6 +53,8 @@
   <script type="text/javascript" src="scripts/codechecker-api/codeCheckerAuthentication.js"></script>
   <script type="text/javascript" src="scripts/codechecker-api/products_types.js"></script>
   <script type="text/javascript" src="scripts/codechecker-api/codeCheckerProductService.js"></script>
+  <script type="text/javascript" src="scripts/codechecker-api/configuration_types.js"></script>
```

Comment:
I think these imports are required only in the `products.html`. You don't want to edit configuration settings for example on the login page.

Reply:
There was a requirement that the notification should appear on every page. I'think without these imports displaying the notification wouldn't work.